### ISSUE TITLE
Google C++ style

### ISSUE THREAD

**Issue by ariddell** [sakrejda/protostan#38]
Is it alright if we standardize on Google's C++ style? https://google.github.io/styleguide/cppguide.html

I find the lack of firm conventions in C++ maddening.

**Comment by sakrejda** [comment]
Could you get the cpplint working?  The Stan (math?) submodule should have the Makefile cpplint rules used in Stan but I couldn't get it going in a sane way last time I tried... without cpplint I have not had good luck trying to follow Google's guide...

**Comment by ariddell** [comment]
Yeah, I'll do this next after I get the POC cleaned up. It's working right now but it needs to be cleaned up and hooked up to travis. I'll turn on cpplint there too.
